### DISCUSSION TITLE
Fan Control won't start

### DISCUSSION THREAD

**Discussion by fshelton79** [Rem0o/FanControl.Releases#2134] [Q&A] [answered]
I have Fan Control in my C: drive and this won't start, but the latest version downloaded and extracted in a different folder starts without issue.  How can I transfer my customizations to the working download or how do I make the already existing install work?

**Comment by mattjones73** [comment]
I updated from 173 to 174, I can see fan control running in task manager, I no longer get my configured temp read out or any icon in the tray to open the app.. 

fshelton, the config is saved as a .json file, typically in the configurations sub directory, you could try to copy it to the new install or import it using Load Configuration and point it to your .json from the old install.

**Comment by Byleth-Eisner** [comment]
I also saw my task tray icon disappeared after updating to 174. I looked into task manager as mentioned by Mattjones73, but couldn't find any trace of fancontrol running there - although it could just be I hadn't looked hard enough. I had a backup of 173 (basically just a drag and drop of the entire FanControl folder) so I just went ahead and deleted my 174 folder and replaced it with the backup, rebooted, and now it seems to be working as like before the update - notwithstanding the update prompt after rebooting.

**Comment by germsf** [comment]
Hello,
22/11/2023 : an update was available to me , when I accepted it, Windows Defender detected a trojan, is this normal? This wasn't the case in the past.
Now the tool won't even run ... I can't find it in running processes ...
![qsd](https://github.com/Rem0o/FanControl.Releases/assets/151723125/0023bf71-632b-45e5-9ccd-37d4498aeb0b)

**Reply by germsf** [reply]
![qsdqsqdq](https://github.com/Rem0o/FanControl.Releases/assets/151723125/245cbb9b-c958-48d7-b02a-c23af8271920)
This is what I had before deleting the file.

**Reply by fshelton79** [reply]
I had the same Defender warning.

**Reply by NeoGee** [reply]
I've exactly the same happen including fan control will no longer run

**Comment by germsf** [comment]
OK, so how I fixed my issue:
1/ backup the userConfig.json
2/ delete the FanControl folder
3/ download the latest archive ( https://github.com/Rem0o/FanControl.Releases ) and all is ok
4/ load the backed up config file , restart the tool, and all works

**Reply by NeoGee** [reply]
Thank you, now have Fan Control back up and running. 
Doing a full virus scan though as shortly after the first trojan flag, I had another warning about Microsoft itself !

**Reply by fshelton79** [reply]
What is the path of that kson file.  Search isn't finding it.

**Reply by germsf** [reply]
C:\Users\xxx\Desktop\FanControl (1)\Configurations\

**Comment by fshelton79** [comment, marked as answer]
I was able to follow another posters suggestion.  I manually downloaded the V174 release and extracted to the same directory telling it to overwrite everything.  This kept my config intact.
